src/server/utils/env.ts

```diff
@@ -0,0 +1,7 @@
+import { config } from "dotenv";
+
+config();
+console.log(process.env.NODE_ENV);
```

Comment:
Add a prefix to this so we know what the print entails in the console

Reply:
Whoops, errant console.log